Added functions to get/set various stockmodel parameters

Feuerbowle opened the pull request:
Added functions to get/set following stockmodel parameters
- Tolerance
- Stepover
- Rest thickness
- Detect overhang
- Workplane

Feuerbowle committed:
Added functions to get/set various stockmodel parameters

Added functions to get/set following stockmodel parameters
- Tolerance
- Stepover
- Rest thickness
- Detect overhang
- Workplane

lukeedw commented on Delcam.ProductInterface.PowerMILL/Classes/PMStockModel.cs:
```diff
@@ -123,6 +126,65 @@ public List<string> States
+        /// Gets or sets the Stepover of the stockmodel.
+        /// </summary>        
+        /// <returns>The Stepover of the stockmodel.</returns>
+        public MM StepOver
```
Please rename to Stepover

lukeedw commented on Delcam.ProductInterface.PowerMILL/Classes/PMStockModel.cs:
```diff
@@ -123,6 +126,65 @@ public List<string> States
+        {
+            get
+            {
+                string workplane = PowerMill.DoCommandEx(string.Format("PRINT PAR \"entity('stockmodel', '{0}').Workplane.Name\"", Name)).ToString();
```
Simplify this by using PRINT PAR TERSE.  Then you don't need the second line to strip off (STRING).

Feuerbowle committed:
Changes as requested

- changed "StepOver" to "Stepover"
- workplane function simpified with "TERSE" command

lukeedw commented:
Thanks. Alpha version should be available to download shortly.
